src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -4624,6 +4624,13 @@ private uint GetValEscapeOfObjectMemberInitializer(BoundExpression expr, uint sc
                 uint receiverEscapeScope = CallingMethodScope;
                 foreach (var escapeValue in escapeValues)
                 {
+                    // This is a call to an indexer so the ref escape scope can only impact the escape value if it 
```

Comment:
```suggestion
                    // This is a call to an indexer so the ref escape scope can only impact the escape value if it
```
Remove trailing whitespace